notebook/static/base/js/utils.js

```diff
@@ -441,17 +441,7 @@ define([
         return txt;
     }
 
-    // Remove chunks that should be overridden by the effect of
-    // carriage return characters
-    function fixCarriageReturn(txt) {
-        var tmp = txt;
-        do {
-            txt = tmp;
-            tmp = txt.replace(/\r+\n/gm, '\n'); // \r followed by \n --> newline
-            tmp = tmp.replace(/^.*\r+/gm, '');  // Other \r --> clear line
-        } while (tmp.length < txt.length);
-        return txt;
-    }
+    var fixCarriageReturn = require('escape-carriage');
```

Comment:
It's been a while since I knew what the state of asset requiring was in the notebook, so I'll let someone else chime in here on how we should do a `require`.